### PR TITLE
Initial commit from datastore-samples

### PR DESCRIPTION
**Pull request by elibixby**
PTAL @GoogleCloudPlatform/devrel-samples 

After this is merged GoogleCloudPlatform/datastore-samples should be safe to delete

### CODE REVIEW

**Comment by elibixby** [comment]
PTAL @jeffmendoza

File: datastore/ndb/overview/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+   gcloud auth login
+   gcloud config set project <your-app-id>
+   ```
+   You don't need a valid app-id to run locally, but will need a valid id to deploy below.
+
+1. Clone this repo.
+
+   ```
+   git clone https://github.com/GoogleCloudPlatform/datastore-pthon-samples.git
+   cd datastore-python-samples/ndb-overview
```
Comment (jeffmendoza) [inline datastore/ndb/overview/README.md]: should be `cd datastore/ndb/overview`

File: datastore/ndb/overview/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+   gcloud components update app
+   gcloud auth login
+   gcloud config set project <your-app-id>
+   ```
+   You don't need a valid app-id to run locally, but will need a valid id to deploy below.
+
+1. Clone this repo.
+
+   ```
+   git clone https://github.com/GoogleCloudPlatform/datastore-pthon-samples.git
```
Comment (jeffmendoza) [inline datastore/ndb/overview/README.md]: should be `git clone https://github.com/GoogleCloudPlatform/python-docs-samples.git`

File: datastore/ndb/transactions/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+   gcloud config set project <your-app-id>
+   ```
+   You don't need a valid app-id to run locally, but will need a valid id to deploy below.
+
+1. Clone this repo.
+
+   ```
+   git clone https://github.com/GoogleCloudPlatform/datastore-samples.git
+   cd datastore-samples/python/ndb/transactions
+   ```
```
Comment (jeffmendoza) [inline datastore/ndb/transactions/README.md]: Same changes needed here

**Comment by elibixby** [comment]
@jeffmendoza I merged the READMEs for the three datastore directories since they were so similar. We can add more subdirectory specific READMEs later if we want.

File: datastore/ndb/transactions/main.py [hunk 4]
```diff
@@ -1,188 +0,0 @@
-# Copyright 2015 Google Inc. All rights reserved.
```
Comment (jeffmendoza) [inline datastore/ndb/transactions/main.py]: Looks like main.py got deleted?

Comment (elibixby) [inline datastore/ndb/transactions/main.py]: Ruhoh was not intended. I'll grab it again

**Comment by jeffmendoza** [comment]
Btw, Looks like you're doing git commits with and email that is not associated with your github account (or any account). That is why it is the grey cat you can't click on.

**Comment by elibixby** [comment]
Yeah, I'm commiting from my google account, because I haven't changed my config from when I was doing Git-on-Borg stuff. Thanks for the heads-up

**Comment by elibixby** [comment]
closing as obsolete

**Pull request closed without merging**